beacon-chain/core/helpers/signing_root.go

```diff
@@ -117,7 +117,11 @@ func VerifyBlockSigningRoot(blk *ethpb.BeaconBlock, pub []byte, signature []byte
 	publicKey := set.PublicKeys[0]
 	root := set.Messages[0]
 
-	if !sig.Verify(publicKey, root[:]) {
+	rSig, err := bls.SignatureFromBytes(sig)
```

Comment:
maybe you can use verify compressed like you do above?

Reply:
I reverted that change, as it can potentially be more expensive in the event our public key is cached.